src/components/ConfigEditor/ConnectionConfig.tsx

```diff
@@ -19,6 +24,13 @@ const ConnectionConfig: React.FC<ConnectionConfigProps> = ({
 }) => {
   const { jsonData, secureJsonData, secureJsonFields } = options;
 
+  // Set some default values
+  useEffect(() => {
+    if (!jsonData.azureCloud) {
+      updateJsonData('azureCloud', azureClouds[0].value);
```

Comment:
I think it would be nice to have these clouds as some stable consts or enums, or from a json file we share with the backend maybe? 👀

I'm just trying to think about how we can eliminate problems with typos and things like that.

Reply:
I agree!